### DISCUSSION TITLE
New Source: SuiteCRM

### DISCUSSION THREAD

**Discussion by csauge** [airbytehq/airbyte#36097] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination? Source SuiteCRM to Postgres
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? No

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? We need to produce KPIs based on SuiteCRM data
* How often do you want to run syncs? daily
* If this is an API source connector, which entities/endpoints do you need supported? SuiteCRM is an open source project based on SugarCRM already managed by Airbyte, so SuiteCRM connector should be very similar (maybe identical) to existing SugarCRM
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? SuiteCRM is open-source and free

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte? Manual integration

**Comment by HowardM12138** [comment]
Hi,

I would like to help. I'm currently picking up the Airbyte code and currently looking at this issue. From SuiteCRM API, I realized that we need an "example.com" domain address to test access. Is there an existing temporary test address that I can use, in addition, username and password to try access to the REST API?

[a comment by an automated account is not shown]
